core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -534,9 +534,10 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 				$pid = $dc->id;
 				$table = $strTable;
 				$ptable = $act != 'edit' ? ($GLOBALS['TL_DCA'][$strTable]['config']['ptable'] ?? null) : $strTable;
-
 				$container = System::getContainer();
 
+				$this->loadDataContainer($ptable);
```

Comment:
`$ptable` can be `null` here, should it fallback to the table instead?

```suggestion
				$this->loadDataContainer($ptable ?? $strTable);
```

Reply:
I don‘t think we have to load the DC at all if `$ptable` is `null`, do we? So maybe we should just wrap it with `if ($ptable)` as in line 577?

Reply:
👍

Reply:
Changed in 7c925bd2ec062900b0b8f39fa41f2acf4a17c251.